src/styles/stars.scss

```diff
@@ -0,0 +1,35 @@
+.card__review-container {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+
+  .stars {
+    display: flex;
+    align-items: center;
+
+    &--4 {
+      display: flex;
+    }
+
+    &__star {
+      width: 16px;
+      height: 16px;
+      margin-right: 4px;
+      background-position: center;
+      background-repeat: no-repeat;
+      background-image: url(../images/star.svg); // Inactive star
+
+      // Active stars based on rating
+      &:nth-child(-n + 4) {
+        background-image: url(../images/star-active.svg);
+      }
+    }
+  }
+
+  .card__review-text {
+    font-family: $font;
```

Comment:
It's great that you're using variables for font-related properties. However, make sure that the `$font` and `$font-weight-standard` variables are defined in your `variables.scss` file. If they are not defined, it will lead to compilation errors.